AudioAnalytics/AudioAnalyzer/src/main/java/sample/AudioAnalyzer.java

```diff
@@ -56,13 +57,15 @@ public static void main(String[] args) {
     private static void runAnalyzer(ConfigWrapper config) {
         // Connect to media services, please see https://docs.microsoft.com/en-us/azure/media-services/latest/configure-connect-java-howto
         // for details.
+        //Add authorityhost to connect to azure china
         TokenCredential credential = new ClientSecretCredentialBuilder()
                 .clientId(config.getAadClientId())
                 .clientSecret(config.getAadSecret())
                 .tenantId(config.getAadTenantId())
+                .authorityHost(AzureAuthorityHosts.AZURE_CHINA)
                 .build();
         AzureProfile profile = new AzureProfile(config.getAadTenantId(), config.getSubscriptionId(),
-                com.azure.core.management.AzureEnvironment.AZURE);
+                com.azure.core.management.AzureEnvironment.AZURE_CHINA);
```

Comment:
Change this to read 

  .authorityHost(AzureAuthorityHosts.AZURE_PUBLIC_CLOUD) // update to use Azure Government or Azure China